notebooks/idr0062_prediction.ipynb

```diff
@@ -12,7 +12,7 @@
     "The image is referenced in the paper \"NesSys: a novel method for accurate nuclear segmentation in 3D\" published August 2019 in PLOS Biology: https://doi.org/10.1371/journal.pbio.3000388 and can be viewed online in the [Image Data Resource](https://idr.openmicroscopy.org/webclient/?show=image-6001247).\n",
     "\n",
     "\n",
-    "In this notebook, the image is loaded together with the labels and analyzed using [StarDist](https://stardist.net/docs/index.html). The StarDist analysis produces a segmentation, which is then viewed side-by-side with the original segmentations produced by the authors of the paper obtained via the loaded labels.\n",
+    "In this notebook, the image is loaded together with the labels and analyzed using [StarDist](https://github.com/stardist/stardist). The StarDist analysis produces a segmentation, which is then viewed side-by-side with the original segmentations produced by the authors of the paper obtained via the loaded labels.\n",
```

Comment:
https://stardist.net/ is still valid

Reply:
I think the github repo is better. See screenshot below. The Stardist readthedocs doc is pointing ultimately to the repo as the source of truth
![Screenshot 2022-05-05 at 14 06 24](https://user-images.githubusercontent.com/2478303/166929262-90cf6571-083a-4534-9144-084e76bbb348.png)